src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,61 @@
-export const PeopleFilters = () => {
+import React from 'react';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../types';
+import { FilterParam } from '../types/FilterParam';
+import { getSearchWith } from '../utils/searchHelper';
+import { SearchLink } from './SearchLink';
+import { Sex } from '../types/Sex';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleFilters: React.FC<Props> = () => {
+  const CENTURIES = ['16', '17', '18', '19', '20'];
```

Comment:
It is better to define this array outside the component, so that it is not created every time during a new rendering.